boards/arm/nrf9160_pca10090/nrf9160_pca10090_common.dts

```diff
@@ -18,39 +18,39 @@
 	leds {
 		compatible = "gpio-leds";
 		led0: led_0 {
-			gpios = <&gpio0 2 GPIO_INT_ACTIVE_HIGH>;
-			label = "Green LED 0";
+			gpios = <&gpio0 2 0>;
```

Comment:
Why not `GPIO_ACTIVE_HIGH`? I thought that was the purpose of these flags with value of 0 - to use them to clearly express what is being configured.

Reply:
Yes, we could add `GPIO_ACTIVE_HIGH` flag here. But to be consistent we would need to add it also to all other DTS files. So far we've been using flags to indicate that settings are different than default, i.e. without flags the pin defaults to active high and output defaults to push pull.